julia_src/utils.jl

```diff
@@ -221,7 +221,7 @@ Base.@kwdef struct Parameter
 	BoilerTechs::Array{String,1}
 	HeatingLoad::Array{Float64,1}
 	CoolingLoad::Array{Float64,1}
-	BoilerEfficiency::Float64
+	BoilerEfficiency::AxisArray
```

Comment:
I have been finding AxisArray's somewhat problematic (such as when assigning a vector of values to them) and so I have been switching them to Dict's. The `BoilerEfficiency` inputs seems like a natural for a Dict: string keys for numeric values. Not necessary to change now (we can do it with v2) so just an FYI.

Reply:
Ok, noted, and yes we can convert with V2 (there are a lot of those for CHP)